We deselect tests/test_distributions.py::test_packages_distributions and so the
coverage is lower than expected.

--- dist-meta-0.9.0/tox.ini.orig
+++ dist-meta-0.9.0/tox.ini
@@ -218,7 +218,7 @@
 [coverage:run]
 
 [coverage:report]
-fail_under = 97
+fail_under = 95
 show_missing = True
 exclude_lines =
     raise AssertionError
